### PR TITLE
Add audio support to mediaIndicator

### LINKED ISSUE #544: Update mediaIndicator to handle audio
Waiting for SVG from UX

**Is your feature request related to a problem? Please describe.**
We currently have a [MediaIndicator](https://github.com/bbc/psammead/blob/latest/packages/components/psammead-media-indicator/src/index.jsx) component, however it only supports video assets. We will need to enable it to display a different icon depending on whether it's a video or an audio asset.

Pass `type` prop of 'audio' or 'video' and switch svg on it. We should also vary the off screen text, and translate it, passing in a value from simorgh.

**Describe the solution you'd like**
Display different SVGs for icons depending on media type.

**Testing notes**
[Tester to complete]

Dev insight: Unit tests, storybook

### PR DESCRIPTION
**Pull request by ghost**
Resolves #544 

**Overall change:** Add `audio` svg and enable switching between `video` and `audio` through a `type` prop
**Code changes:**

- Add `mediaIcons` file to hold the svgs
- Add `audio` svg
- Add `type` prop which expects one of two values `audio` or `video` and defaults to `video`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-media-indicator/src/index.stories.jsx [hunk 1]
```diff
@@ -20,17 +20,44 @@ storiesOf('MediaIndicator', module)
   .addDecorator(dirDecorator)
   .add(
     'default',
```
Comment (dr3) [inline packages/components/psammead-media-indicator/src/index.stories.jsx]:
```suggestion
    'video without duration',
```

**Review by dr3 — DISMISSED** [review]

**Review by DenisHdz — DISMISSED** [review]
alt-text

**Comment by david-boydell** [comment]
Hi Isaac, I'm not sure why I didn't spot this before but the height of the media indicator is two px fewer on the "_without duration_" than on the "_with duration_" due to the presence of the text span. Is there anything we can do about this?

Cheers
David

**Review by DenisHdz — APPROVED** [review]
LGTM 👍

**Review by dr3 — APPROVED** [review]
Setting the height to 2rem isnt ideal, as it doesnt scale with the font if `GEL_MINION` was to change. For now though, LGTM

**Comment by david-boydell** [comment]
Great, thanks Isaac, this is ready for merge, good stuff.

**Pull request merged**